src/components/MoneyTemporaryForRefactorRequestConfirmationList.tsx

```diff
@@ -774,17 +762,24 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
             shouldShow: shouldShowCategories,
             isSupplementary: !isCategoryRequired,
         },
-        ..._.map(policyTagLists, ({name}, index) => ({
+        ...policyTagLists.map(({name}, index) => ({
             item: (
                 <MenuItemWithTopDescription
                     key={name}
                     shouldShowRightIcon={!isReadOnly}
-                    title={TransactionUtils.getTagForDisplay(transaction, index)}
+                    title={TransactionUtils.getTagForDisplay(transaction ?? null, index)}
```

Comment:
I think we don't have to default to null for transactions, wdyt? Same for other places in this file
```suggestion
                    title={TransactionUtils.getTagForDisplay(transaction, index)}
```